app/containers/SongDetailContainer/tests/index.test.js

```diff
@@ -0,0 +1,90 @@
+/**
+ *
+ * Tests for SongDetail container
+ *
+ *
+ */
+
+import React from 'react';
+import { renderProvider, timeout } from '@utils/testUtils';
+import { SongDetailContainerTest as SongDetailContainer, mapDispatchToProps } from '../index';
+import { songDetailContainerTypes } from '../reducer';
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: jest.fn(() => ({ trackId: 12345 }))
+}));
+
+describe('<SongDetailContainer /> container tests', () => {
+  let submitSpy;
+
+  beforeEach(() => {
+    submitSpy = jest.fn();
+    //jest.spyOn(Router, 'useParams').mockReturnValue({ trackId: 12345 });
```

Comment:
please remove the comment

Reply:
Will try out spyOn method later on that's why kept it.